JSDemos/Demos/DataGrid/DeferredSelection/description.md

```diff
@@ -1 +1,12 @@
-The **DataGrid** widget supports deferred row selection. When it&nbsp;is&nbsp;enabled, the **DataGrid** does not make requests for data until this is&nbsp;demanded from the API. In&nbsp;this demo, data is&nbsp;loaded when the jQuery Deferred object returned by&nbsp;the **getSelectedRowsData()** method is&nbsp;resolved. To&nbsp;specify initially selected rows, the **selectionFilter** option is&nbsp;used. Use deferred selection mode when working with a&nbsp;huge amount of&nbsp;remote data.
\ No newline at end of file
+If you enable deferred row selection, the grid does not request selected rows' data with every selection change. For example, if a user clicks the checkbox in the column header to select all the rows, the grid does not immediately fetch all data from the server. This is helpful in the following cases:
+
+- You process data on the server and do not want to load the selected rows' data.
+- You do process selected records on the client, but want to reduce the number of requests that are sent.
```

Comment:
Is the word 'do' really required here?

>>>number of requests that are sent
Maybe change it to the 'requests to the server' in order to avoid passive voice?